src/mocks/mock-test-result.ts

```diff
@@ -40,7 +41,15 @@ const mockTestTypeList = (numberOfItems: number = 1) =>
       },
       modificationTypeUsed: 'modifications number ' + Math.round(Math.random() * 1000).toString(),
       particulateTrapFitted: 'particulate trap ' + Math.round(Math.random() * 1000).toString(),
-      particulateTrapSerialNumber: 'ABC' + Math.round(Math.random() * 1000).toString()
+      particulateTrapSerialNumber: 'ABC' + Math.round(Math.random() * 1000).toString(),
+
+      testStationName: 'Abshire-Kub',
+      testStationPNumber: '(P12346)',
```

Comment:
nitpick - should the PNumber be in brackets? (I don't think I've seen them stored like this in the database)

Reply:
I was just mimicking the mockup we had been given for this, my goal was to get the data to reflect it as best I could before finishing it just to be sure, i'm fairly certain it'll be different in the final design anyway